src/views/Home.vue

```diff
@@ -1,93 +1,187 @@
 <template>
-  <div class="columns is-multiline has-text-centered px-0 mx-0">
+  <div class="columns is-multiline has-text-centered px-0 mx-0 home__container">
     <div class="column is-full box py-6 px-4 recipe-app__content">
-      <h1 class="title is-2 is-uppercase is-spaced">
+      <h1 class="title is-1 is-uppercase is-spaced ">
         Welcome to the recipe app
       </h1>
-      <h2 class="subtitle is-3">
+      <h2 class="subtitle is-2  ">
         This app will help you save your favorite recipes fast
       </h2>
       <button
-        class="button is-large is-primary m-2"
-        @click="taskStore.showmodal = true"
+        class="button is-large is-primary m-2 home__button"
+        @click="recipesStore.showmodal = true"
       >
         Add new recipe
       </button>
       <button
-        class="button is-large is-link m-2"
-        @click="taskStore.showAll = !taskStore.showAll"
+        class="button is-large is-link m-2 home__button "
+        @click="recipesStore.showAll=!recipesStore.showAll"
```

Comment:
It is better for a user to see the new recipe as it was created without extra moves (no need to click a button to show all recipes)